src/win/process.c

```diff
@@ -569,7 +631,20 @@ int make_program_args(char** args, int verbatim_arguments, WCHAR** dst_ptr) {
   }
 
   pos = dst;
-  for (arg = args; *arg; arg++) {
+  arg = args;
+  /* If launching script file, add 'cmd /C scriptname',
+     and ignore first arg */
+  if (script_file != NULL) {
+    /* advance arg pointer if args[0] was set properly */
+    if(*arg != NULL) arg++;
+    wcscpy(pos, L"cmd /c");
```

Comment:
Unfortunately `cmd /c` behaves weirdly if the command line starts and ends with a quote (e.g. `"c:\program files\bar.bat" 1 2 "hello world"`. We would need a way to work around that.